applications/GeoMechanicsApplication/custom_workflows/dgeosettlement.cpp

```diff
@@ -183,6 +198,21 @@ void KratosGeoSettlement::AddDegreesOfFreedomTo(Kratos::ModelPart &rModelPart)
     VariableUtils().AddDof(VOLUME_ACCELERATION_Z, rModelPart);
 }
 
+LoggerOutput::Pointer KratosGeoSettlement::CreateLoggingOutput(std::stringstream& rKratosLogBuffer)
+{
+    LoggerOutput::Pointer logger_output(new LoggerOutput(rKratosLogBuffer));
```

Comment:
Here we can use std::make_shared instead of new
```suggestion
    auto logger_output = std::make_shared<LoggerOutput>(rKratosLogBuffer));
```